src/components/About/About.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import {
   Breadcrumbs,
-  Footer,
+  // Footer,
```

Comment:
Is there a reason we are commenting these out instead of removing them and removing our Footer component completely?

Reply:
Initially this was a test branch so I left them commented in case we wanted to revert. Removed the commented out components/imports.